server/middleware/__tests__/configureEstablishment.spec.js

```diff
@@ -0,0 +1,41 @@
+const { configureEstablishment } = require('../configureEstablishment');
+
+describe('configureEstablishment', () => {
+  const next = jest.fn();
+
+  beforeEach(() => {
+    next.mockClear();
+  });
+
+  it('should use the set the session from the config', () => {
+    const req = {
+      session: {
+        establishmentName: 'wayland',
+        establishmentId: 793,
+      },
+    };
+
+    const res = { locals: {} };
```

Comment:
Pop that `const res = { locals: {} };` in the beforeEach

Reply:
I almost did this :-D